configs/foreman/nightly.yaml

```diff
@@ -98,3 +98,271 @@
       foreman-plugins-nightly-rhel7-dist:
         foreman-plugins-nightly-rhel7: 2
         foreman-plugins-nightly-nonscl-rhel7: 0
+  - name: foreman-client-nightly-rhel5
+    based_off: null
+    helper_tags:
+      foreman-client-nightly-rhel5-override: null
+      foreman-client-nightly-rhel5: null
+    build_target: foreman-client-nightly-rhel5-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-client-nightly-rhel5-build:
+        foreman-client-nightly-rhel5-override: 0
+      foreman-client-nightly-rhel5-override:
+        foreman-client-nightly-rhel5: 0
+      foreman-client-nightly-rhel5: {}
+    external_repos:
+      - rhel-5.9-server
+      - epel-5
+    build_groups:
+      build:
+        - bash
+        - bzip2
+        - coreutils
+        - cpio
+        - diffutils
+        - findutils
+        - gawk
+        - gcc
+        - gcc-c++
+        - grep
+        - gzip
+        - info
+        - make
+        - patch
+        - redhat-release-server
+        - redhat-rpm-config
+        - rpm-build
+        - sed
+        - shadow-utils
+        - tar
+        - unzip
+        - util-linux
+        - which
+        - xz
+      srpm-build:
+        - bash
+        - redhat-release-server
+        - redhat-rpm-config
+        - rhpkg-simple
+        - rpm-build
+        - shadow-utils
+  - name: foreman-client-nightly-rhel6
+    based_off: null
+    helper_tags:
+      foreman-client-nightly-rhel6-override: null
+      foreman-client-nightly-rhel6: null
+    build_target: foreman-client-nightly-rhel6-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-client-nightly-rhel6-build:
+        foreman-client-nightly-rhel6-override: 0
+      foreman-client-nightly-rhel6-override:
+        foreman-client-nightly-rhel6: 0
+      foreman-client-nightly-rhel6: {}
+    external_repos:
+      - rhel-6.6-server
+      - rhel-6.6-server-optional
+      - epel-6
+    build_groups:
+      build:
```

Comment:
Are these packages that should be used to build packages for the tags or are they supposed to already be tagged into the tags?

Reply:
Both. You can define buildroot group packages that get injected into the buildroot before any other actions take place. These are often common to every build packages such ass tar or gzip as well as packages that have RPM macros that need to be defined before a package starts to get built.